v03_pipeline/lib/tasks/reference_data/update_variant_annotations_table_with_updated_reference_dataset_test.py

```diff
@@ -412,21 +376,10 @@ def test_update_vat_mito_38(
     @patch(
         'v03_pipeline.lib.tasks.base.base_update_variant_annotations_table.BaseUpdateVariantAnnotationsTableTask.initialize_table',
     )
-    @patch(
-        'v03_pipeline.lib.tasks.base.base_update_variant_annotations_table.UpdatedReferenceDatasetTask',
-    )
-    @patch(
-        'v03_pipeline.lib.tasks.base.base_update_variant_annotations_table.UpdatedReferenceDatasetQueryTask',
-    )
```

Comment:
were these mocks just not needed?

Reply:
Sorta!  The `UpdatedReferenceDataset` are no-ops if every reference dataset is on disk, which they now are (there were maybe 2-3 missing).  The `UpdatedReferenceDatasetQuery` now just run!  It's a slight performance detriment to the tests though.